sdk/playwrighttesting/microsoft-playwright-testing/review/microsoft-playwright-testing.api.md

```diff
@@ -4,6 +4,7 @@
 
 ```ts
 
+import type { PlaywrightTestConfig } from '@playwright/test';
```

Comment:
does it need playwright in dependency for this package ?

Reply:
https://github.com/Azure/azure-sdk-for-js/blob/a2ca1203f900d146d4054508c1aebfebb762e24f/sdk/playwrighttesting/microsoft-playwright-testing/package.json#L81

We already have dev dependency on playwright.

Reply:
If you're using it in source code, even just the types, I believe you'll need `@playwright/test` to be declared under `dependencies` (or peerDeps) - `devDependencies` is not sufficient in this case. 

What is the min-version of `@playwright/test` that your service supports? Is that documented somewhere? Can we just revert to supporting a single globalSetup file for now which should work for all versions? That'll give us some time to better work through the API design instead of further tightly-coupling our types to the underlying playwright types (which may of course evolve)

Reply:
- Min version we support is 1.47 , its documented and sdk throw error for lower version , https://github.com/Azure/azure-sdk-for-js/tree/881ef219d29e17bd661655baf16ea4993d9c4722/sdk/playwrighttesting/microsoft-playwright-testing
- reverting to single globalsetup is breaking for 1.49 customer since oss has changed the type change in 1.49, currently sdk is broken for many customers. 
- so far we have two solutions either use Any or use types from OSS, we discussed it with OSS (Max) and it looks good to them.

Reply:
Ok thanks for letting me know. I approved the PR pre-emptively but I do think you need to include @playwright/test as a `dependency`

Reply:
added peer dependency on playwright